.github/copilot-instructions.md

```diff
@@ -0,0 +1,16 @@
+Follow GLPI’s latest coding standards and best practices: naming, indentation, comments, and PSR-12 compliance.
```

Comment:
```suggestion
Follow GLPI’s latest coding standards and best practices: naming, indentation, comments, and PER Coding Style 3.0 compliance.
```

PER 3 has replaced PSR 12 for a while now.

Reply:
It should just respect our current CI tools (php-cs-fixer, phpstan & rector).